StripeUICore/StripeUICore/Source/Elements/Factories/Address/AddressSectionElement.swift

```diff
@@ -16,12 +16,14 @@ import UIKit
  In addition to the physical address, it can collect other related fields like name.
  */
 @_spi(STP) public class AddressSectionElement: ContainerElement {
+    public typealias DidUpdateAddress = (AddressDetails) -> Void
```

Comment:
Why the typealias? The naming doesn't feel Swifty. Can we just use the closure type directly?

Reply:
It's following the naming convention from [here](https://github.com/stripe/stripe-ios/blob/3edcd2f3/StripeUICore/StripeUICore/Source/Elements/DropdownFieldElement.swift#L20), and I'm using a typealias to follow the same convention.

Reply:
I was originally thinking this was public API, but just noticed it's `@_spi`, so I'm less concerned here. Still, we should probably look at improving this at some point. The naming convention is very Java, and I'm not convinced the typealiases add value since we're not referencing them in a bunch of places